File: bench/tasks/inventory/task.js
import fs from "node:fs";
const old = JSON.parse(fs.readFileSync("old.json", "utf8"));
const cur = JSON.parse(fs.readFileSync("new.json", "utf8"));
for (const k in cur) if (!(k in old)) console.log("+", k, cur[k]);
for (const k in old) if (!(k in cur)) console.log("-", k);
for (const k in old) if (k in cur && cur[k] !== old[k]) console.log(`~ ${k} ${old[k]}->${cur[k]}`);

File: bench/tasks/inventory/task.py
import json

old = json.load(open("old.json"))
new = json.load(open("new.json"))
for k in new:
    if k not in old:
        print("+", k, new[k])
for k in old:
    if k not in new:
        print("-", k)
for k in old:
    if k in new and new[k] != old[k]:
        print(f"~ {k} {old[k]}->{new[k]}")
